server/src/yml-objects/YmlEnumMember.ts

```diff
@@ -0,0 +1,18 @@
+import { CompletionItemKind } from 'vscode-languageserver';
+
+import { AbstractYmlObject, YmlType } from './AbstractYmlObject';
+
+export class YmlEnumMember extends AbstractYmlObject {
+    public identifier: string;
+    public domains: YmlType;
+    public readonly kind = CompletionItemKind.EnumMember;
+
+    constructor(indentifier: string, uri: string) {
+        super(indentifier, CompletionItemKind.EnumMember, uri);
+    }
+
+    public getShortName() {
+        const functionNameSubParts = this.label.split('::');
```

Comment:
I thought it could also be a single `:`? I’m always lost with that stuff.

Reply:
One single `:` is just meaningless but is used to mark some fake namespaces.
The two `:` in `Class::methodName` or `Enum::enumElement` split classes and members.
There is also a triple `:` variant for something I can't remember. 
Anyway, it is correct here.

Reply:
ok thanks